src/main/java/io/littlehorse/quickstart/QuickstartWorkflow.java

```diff
@@ -1,30 +1,36 @@
 package io.littlehorse.quickstart;
 
+import io.littlehorse.sdk.wfsdk.NodeOutput;
 import io.littlehorse.sdk.wfsdk.WfRunVariable;
 import io.littlehorse.sdk.wfsdk.Workflow;
 import io.littlehorse.sdk.wfsdk.WorkflowThread;
 
 public class QuickstartWorkflow {
 
-    public static final String WF_NAME = "quickstart";
-    public static final String GREET = "greet";
-
     /*
      * This method defines the logic of our workflow
      */
     public void quickstartWf(WorkflowThread wf) {
         // Create an input variable, make it searchable
-        WfRunVariable name = wf.declareStr("input-name").searchable();
+        WfRunVariable firstName = wf.declareStr("firstName").searchable().required();
+        WfRunVariable lastName = wf.declareStr("lastName").searchable().required();
+        WfRunVariable ssn = wf.declareInt("ssn").masked().required();
+
+        // Call the verify-identity task and retry it up to 3 times if it fails
+        wf.execute("verify-identity", firstName, lastName, ssn).withRetries(3);
+
+        // This is a blocking call, so it will wait until the event is posted or
+        // if the timeout is reached
+        NodeOutput identityPosted = wf.waitForEvent("identity-verified").timeout(60 * 60 * 24 * 3);
```

Comment:
Let's use mutations and conditionals here rather than just passing the output directly into the task.

It might even make sense to have three `TaskDef`s, one for `accept-customer` and `reject-customer`. Either way, the conditional should be shown in the `WfSpec` not by just a boolean flag. There is also no type checking here.